tests/golden.rs

```diff
@@ -74,6 +74,24 @@ fn encode_ic11() {
     encoder_test("32x32.png", IconType::RGBA32_16x16_2x, "ic11.icns");
 }
 
+#[test]
+fn decode_ic13() {
+    decoder_test("ic13.icns", IconType::RGBA32_128x128_2x, "256x256.png");
+}
+
+#[test]
+fn decode_ic08(){
```

Comment:
This test seems to be failing for me when I run it locally.  (Hmm, looks like I need to set up CI for this repo.)

Reply:
@0rvar It seems the Loom icon added in the test case is ic09 hence why the test-case passes for it. 
The ic08 however happens to be of type is32, which includes the jpeg2000 images.
<img width="175" alt="Screenshot 2024-09-22 at 12 47 01 AM" src="https://github.com/user-attachments/assets/b82c36f0-6c33-4c9b-9525-b0ab8acb82db">
<img width="817" alt="Screenshot 2024-09-22 at 12 49 41 AM" src="https://github.com/user-attachments/assets/23c6811e-9fc9-4087-8252-4e14d8208673">

Reply:
I'm not totally sure why, given the JP2000 decoding is working (which I'm not totally sure of as I tested it with a jp2000 file which I created of type ic10 and it didn't seem to function) 
I believe that the test should've passed but I also have not spent much time in this repo overall.

Reply:
I just now enabled CI testing for this repo.  If you rebase, we should be able to see the results of that.